### DISCUSSION TITLE
Picorv32a chip size

### DISCUSSION THREAD

**Discussion by giumaug** [The-OpenROAD-Project/OpenLane#1441] [General]
I completed OpenLane full flow on picorv32a project provided in the designs folder.
I have a doubt regarding the chip size.
Under reports/floorplan/3-initial_fp_die_area.rpt I found 
0.0 0.0 545.17 555.89.
Looking on metric.csv I found 0.3030545513mm^2 that sound like the product of 545.17 x 555.89.
My questions: 
1) What is the measure unit of these numbers ?
2) Are 544.17 and 555.89 millimeters ?

**Comment by vijayank88** [comment]
@giumaug 
`Are 544.17 and 555.89 millimeters ?`
This measured on Micron.

Final die size  as mention in mm^2 in `metrics.csv`
